packages/hash/frontend/src/pages/[account-slug]/types/entity-type/multiple-values-cell.tsx

```diff
@@ -0,0 +1,162 @@
+    }
+    // Should only be triggered when maxValue updates
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [propertyIndex, setValue, maxValue]);
+
+  return (
+    <>
+      <TableCell
+        ref={(ref: HTMLDivElement) => setAnchorEl(ref)}
+        sx={({ palette, transitions }) => ({
+          px: "0px !important",
+          position: "relative",
+          cursor: "pointer",
+          textAlign: "center",
+          transition: transitions.create("border-color"),
+          border: `1px solid ${
+            multipleValuesMenuOpen ? palette.gray[40] : "transparent"
+          } !important`,
+          "&:hover": {
+            borderColor: `${palette.gray[40]} !important`,
+          },
+        })}
+        onClick={() => setMultipleValuesMenuOpen(true)}
+      >
+        <Box
+          sx={({ palette, transitions }) => ({
+            display: "inline-flex",
+            borderRadius: "4px 30px 30px 4px",
+            backgroundColor: "transparent",
+            transition: transitions.create(["padding", "background-color"]),
+            ...(array && !multipleValuesMenuOpen
+              ? {
+                  py: 0.5,
+                  px: 0.75,
+                  background: palette.gray[20],
+                }
+              : {}),
+          })}
+        >
+          <Checkbox
+            sx={{
+              zIndex: 1,
+            }}
+            checked={array}
+            {...register(`properties.${propertyIndex}.array`)}
+          />
+          <Collapse
+            orientation="horizontal"
+            in={array && !multipleValuesMenuOpen}
+          >
+            <Typography
+              variant="smallTextLabels"
+              sx={{
+                display: "flex",
+                ml: 1,
+                fontWeight: 500,
+                whiteSpace: "nowrap",
+                color: ({ palette }) => palette.gray[70],
+              }}
+            >
+              {minValue !== maxValue ? `${minValue} to ${maxValue}` : minValue}
+            </Typography>
+          </Collapse>
+        </Box>
+      </TableCell>
+
+      <Popper
+        open={multipleValuesMenuOpen}
+        anchorEl={anchorEl}
+        container={anchorEl}
+        placement="bottom"
+        sx={{ width: 1, zIndex: 1 }}
+        transition
+      >
+        {({ TransitionProps }) => {
+          return (
+            <ClickAwayListener
+              onClickAway={() => setMultipleValuesMenuOpen(false)}
+            >
+              <Fade {...TransitionProps}>
+                <Box
+                  sx={({ palette }) => ({
+                    border: 1,
+                    p: 1.5,
+                    background: palette.white,
+                    borderColor: palette.gray[30],
+                  })}
+                >
+                  <TextField
+                    type="number"
+                    size="small"
+                    label="Minimum"
+                    value={minValue}
```

Comment:
You don't need value when using `register`.

Reply:
Removed in 8a0a1d6d73c61b0e63555bf0abfb84f0807a83fd.